aten/src/ATen/WrapDimUtilsMulti.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "ATen/TensorImpl.h"
+#include "ATen/WrapDimUtils.h"
+#include <sstream>
+#include <bitset>
+
+namespace at {
+
+// This is in an extra file to work around strange interaction of
+// bitset on Windows with operator overloading
+
+constexpr size_t dim_bitset_size = 64;
+
+static inline std::bitset<dim_bitset_size> dim_list_to_vector(IntList dims, int64_t ndims, bool wrap_scalar=true) {
```

Comment:
Perhaps we could rename this to something more appropriate? Something like `mark_reduce_dims`?

Reply:
Good catch, `vector` is misleading. But would something more like a conversion function `dim_list_to_bitset` be OK? I use the function for other dimension lists, too (e.g. expansion), so I would not necessarily use reduce as a name. (I currently use for expansion and reduction it while reworking the bilinear fix and also for the pairwise reduction for `einsum` - though there it is reduction...)

Reply:
Ok, let’s go with your name